lemur/manage.py

```diff
@@ -188,216 +187,308 @@ def generate_settings():
-        sys.stdout.write(
-            "[!] Using {0} as specified by LEMUR_SECURITY_TEAM_EMAIL for notifications\n".format(
-                recipients
-            )
+    else:
+        click.echo(
+            "[-] Default user has already been created, skipping...!\n"
         )
-        notification_service.create_default_expiration_notifications(
-            "DEFAULT_SECURITY", recipients=recipients
+
+    intervals = current_app.config.get(
+        "LEMUR_DEFAULT_EXPIRATION_NOTIFICATION_INTERVALS", []
+    )
+    click.echo(
+        "[!] Creating {num} notifications for {intervals} days as specified by LEMUR_DEFAULT_EXPIRATION_NOTIFICATION_INTERVALS".format(
+            num=len(intervals), intervals=",".join([str(x) for x in intervals])
         )
+    )
 
-        _DEFAULT_ROTATION_INTERVAL = "default"
-        default_rotation_interval = policy_service.get_by_name(
-            _DEFAULT_ROTATION_INTERVAL
+    recipients = current_app.config.get("LEMUR_SECURITY_TEAM_EMAIL")
+    click.echo("[+] Creating expiration email notifications!")
+    click.echo(
+        "[!] Using {0} as specified by LEMUR_SECURITY_TEAM_EMAIL for notifications".format(
+            recipients
         )
+    )
+    notification_service.create_default_expiration_notifications(
+        "DEFAULT_SECURITY", recipients=recipients
+    )
 
-        if default_rotation_interval:
-            sys.stdout.write(
-                "[-] Default rotation interval policy already created, skipping...!\n"
-            )
-        else:
-            days = current_app.config.get("LEMUR_DEFAULT_ROTATION_INTERVAL", 30)
-            sys.stdout.write(
-                "[+] Creating default certificate rotation policy of {days} days before issuance.\n".format(
-                    days=days
-                )
+    _DEFAULT_ROTATION_INTERVAL = "default"
+    default_rotation_interval = policy_service.get_by_name(
+        _DEFAULT_ROTATION_INTERVAL
+    )
+
+    if default_rotation_interval:
+        click.echo(
+            "[-] Default rotation interval policy already created, skipping...!\n"
+        )
+    else:
+        days = current_app.config.get("LEMUR_DEFAULT_ROTATION_INTERVAL", 30)
+        click.echo(
+            "[+] Creating default certificate rotation policy of {days} days before issuance.".format(
+                days=days
             )
-            policy_service.create(days=days, name=_DEFAULT_ROTATION_INTERVAL)
+        )
+        policy_service.create(days=days, name=_DEFAULT_ROTATION_INTERVAL)
 
-        sys.stdout.write("[/] Done!\n")
+    click.echo("[/] Done!")
 
 
-class CreateUser(Command):
+@cli.command("create_user")
+@click.option("-u", "--username", "username", required=True)
+@click.option("-e", "--email", "email", required=True)
+@click.option("-a", "--active", "active", type=bool, default=True, show_default=True)
+@click.option("-r", "--roles", "roles", multiple=True, required=True)
+@click.option("-p", "--password", "password", required=True)
```

Comment:
Should we remove required constraint? otherwise, I'm not sure we'll ever hit the `if not password` conditional